contracts/FlowIDTableStaking.cdc

```diff
@@ -719,7 +719,7 @@ pub contract FlowIDTableStaking {
                 if nodeRecord.tokensStaked.balance == 0.0 { continue }
 
                 /// Calculate the amount of tokens that this node operator receives
-                let rewardAmount = rewardsForNodeTypes[nodeRecord.role]! * (nodeRecord.tokensStaked.balance / FlowIDTableStaking.totalTokensStakedByNodeType[nodeRecord.role]!)
+                let rewardAmount = FlowIDTableStaking.epochTokenPayout * (nodeRecord.tokensStaked.balance / totalStaked)
```

Comment:
`nodeRecord.tokensStaked.balance` does _not_ include any delegated tokens, right?

Reply:
Yes, that is correct. 👌